### PR TITLE
Add minimum version info for External Policies

### PR DESCRIPTION
**Pull request by prashantkalkar**
Added kops version information for external policies support.

### CODE REVIEW

File: docs/iam_roles.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ iam:
 
 ## Adding External Policies
 
+Since: kops version 1.18.x
```
Comment (olemarkus) [inline docs/iam_roles.md:74]: See the section above for an example of how to add version information.

Comment (hakman) [inline docs/iam_roles.md:74]:
```suggestion
{{ kops_feature_table(kops_added_default='1.18') }}
```

**Comment by hakman** [comment]
@prashantkalkar please sign the CLA to be able to merge this. Thanks!

**Comment by prashantkalkar** [comment]
@hakman I have now signed the CLA (I was facing some issues earlier).

**Comment by hakman** [comment]
No worries. Thanks @prashantkalkar.
/ok-to-test
/lgtm
/approve

**Pull request merged**